### PR TITLE
Km snm3c overview doc

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

Updating snM3 Overview doc

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]

File: website/docs/Pipelines/snM3C/README.md [hunk 1]
```diff
@@ -6,87 +6,182 @@ slug: /Pipelines/snM3C/README
...
+The Single Nucleus Methly-Seq and Chromatin Capture (snM3C) workflow is an open-source, cloud-optimized computational workflow for processing single-nucleus methylome and chromatin contact (snM3C) sequencing data. The workflow is designed to demultiplex and align raw sequencing reads, call chromatin contacts, and generate summary metrics. 
 
-## Set-up
+The workflow is developed in collaboration Hanqing Liu and the laboratory of Joseph Ecker. For more information about the snM3C tools and analysis, please see the [YAP documentation](https://hq-1.gitbook.io/mc/) or the [cemba_data](https://github.com/lhqing/cemba_data) GitHub repository created by Hanqing Liu.
```
Comment (nikellepetrillo) [inline website/docs/Pipelines/snM3C/README.md:16]:
```suggestion
The workflow is developed in collaboration with Hanqing Liu and the laboratory of Joseph Ecker. For more information about the snM3C tools and analysis, please see the [YAP documentation](https://hq-1.gitbook.io/mc/) or the [cemba_data](https://github.com/lhqing/cemba_data) GitHub repository created by Hanqing Liu.
```

File: website/docs/Pipelines/snM3C/README.md [hunk 2]
```diff
@@ -6,87 +6,182 @@ slug: /Pipelines/snM3C/README
...
+| r1_left_cut | Optional integer describing the number of bases to be trimmed from the beginning of read 1 with Cutadapt; default is 10. |
+| r1_right_cut | Optional integer describing the number of bases to be trimmed from the end of read 1 with Cutadapt; default is 10. |
+| r2_left_cut | Optional integer describing the number of bases to be trimmed from the beginning of read 2 with Cutadapt; default is 10. |
+| r2_right_cut | Optional integer describing the number of bases to be trimmed from the end of read 2 with Cutadapt; default is 10. |
+| min_read_length | Optional integer; if a read length is smaller than `min_read_length`, both paired-end reads will be discarded; default is 30.  |
+| num_upstr_bases | Optional integer describing the number of bases upstream of the C base to include in ALLC file context column created using ALLCools; default is 0. |
+| num_downstr_bases | Optional integer describing the number of bases upstream of the C base to include in ALLC file context column created using ALLCools; default is 2. |
```
Comment (nikellepetrillo) [inline website/docs/Pipelines/snM3C/README.md:68]: Not super familiar with this input, but should it be "number of bases downstream of the C base" ?

**Review by nikellepetrillo — APPROVED** [review]
thanks kaylee!

File: website/docs/Pipelines/snM3C/README.md (hunk 2 above)
Comment (kayleemathews) [reply website/docs/Pipelines/snM3C/README.md:68]: Yes good catch! Thank you!

**Pull request merged**